app/jobs/ingest_pulfa_job.rb

```diff
@@ -40,6 +39,18 @@ def ingest
       r.save!
     end
 
+    def work_attributes
+      {
+        title: [@mets.xpath("//mets:structMap/mets:div/@LABEL").first.value],
```

Comment:
This is only for the migration of the legacy content, right (because we won't have METS going forward)